### ISSUE TITLE
Classes and Controlers from SRC directory do not get installed

### ISSUE THREAD

**Issue by sometechguy** [bjuppa/laravel-blog#14]
I installed it on an existing site and went through the steps listed on this git.  However, neither the classes nor the controllers got copied.  I was able to run the migration and I can see the new routes too.

**Comment by bjuppa** [comment]
Morning, and glad to hear you're trying out my package! I'm actually planning to release `1.0` this week, so you'll have something suitable for a live site soon!

(Although the admin panel for the blog system will be released as a separate, optional package, and that will take a few more weeks to release. So until then you'll have to code your own back end for editing blog entries in the database)

But, back to your issue... If you were able to run the migration and also can see the blog routes, that's a good indication everything has been installed. You can check in the directory `vendor/bjuppa/laravel-blog`, that's where `composer` would put the source code from this package after install, can you find them there? The controllers does not need copying into the Laravel app directory, they're used from the package's src directory.

**Comment by sometechguy** [comment]
i was able to see route only if I ran the artisan routes command.  I wasn't able to see them in the routes directory and it even had broken the login access to to my original site.  I spent 3 days trying to use various Laravel blog packages including yours which I gave a shot to twice in this time period.  Finally, I have gone back to Drupal, a platform I was very comfortable with before I picked up Laravel 3 years ago.   My motivation to use Laravel was because it was easier for me to use JWT protected webservices and easily build my custom logic in the controllers with it and now I wanted to extend Laravel for blogging as well so that I don't have to manage two systems.  But looking at the effort required to just to build or integrate a simple functionality something Taylor Otwell should have made available out of box I am finding it easier to just blog with Drupal.    I will reconsider your package once it is more stable.  Thank you for your help.
